patches/build.gen.py.patch

```diff
@@ -32,7 +32,15 @@ index 9168035b..7e420980 100755
      if is_gcc(cxx):
        cflags.append('-Wno-redundant-move')
        # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=104336
-@@ -905,7 +908,7 @@ def WriteGNNinja(path, platform, host, options, args_list):
+@@ -536,6 +539,7 @@ def WriteGNNinja(path, platform, host, options, args_list):
+       cflags.append('-D_OPEN_SYS_FILE_EXT')
+       cflags.append('-DPATH_MAX=1024')
+       cflags.append('-DZOSLIB_OVERRIDE_CLIB')
++      ldflags.append('-m64')
```

Comment:
Suggest also adding `cflags.append('-m64')` - in case CC/CXX don't specify the 64-bit versions (ibm-clang64/ibm-clang++64).

Reply:
My apologies, gaby, I overlooked this.  I'll redo another patch to address it.